standard.md

```diff
@@ -0,0 +1,136 @@
+- **ensemble**: `str`. Ensemble identifier of a dataset.
+
+#### Related to the variable
+
+- **table**: `str`. Variable's original MIP table.
+- **original_frequency**: `str`. Variable's original frequency.
+- **frequency**: `str`. Variable's frequency when entering the diagnostic.
+- **modeling_realm**: `str`. Realms a variable belongs to.
+- **grid**: `str`. Variable's original grid.
+- **units**: `str`. Variable's units.
+- **short_name**: `str`. Variable's name used in the file as given by CF conventions.
+- **standard_name**: `str`. Standard name of the variable as given by CF conventions.
+- **long_name**: `str`. Long name of the variable as given by CF conventions.
+
+#### Others
+
+- **start**: `str`. Starting date of the data, given in ISO 8601 format for dates (YYYYMMDD).
+- **end**: `str`, Ending date of the data, in ISO 8601 format for dates (YYYYMMDD).
+- **start_year**: `int`. Starting year of the data.
+- **end_year**: `int`. End year of the data.
+- **reference_dataset**: `str`. Alias of the dataset to be used as the reference for comparisons.
+
+## The script formal description file
+
+It is *recommended* that each script comes with a description file in YAML syntax which includes information regarding: documentation, configuration of script, and optional settings.
+
+The following labels should be provided in order to complete the documentation section:
+
+- **title**: `str`. Label for the title of the diagnostic.
+- **description** `str`. A short description of the diagnostic.
+- **references** `list(str)`. List of references to be cited if using the diagnostic.
+- **authors** `list(str)`. List of authors that developed the diagnostic.
+- **read_more**: `url`. An URL to the online documentation.
+
+The labels listed below should be provided in order to complete the configuration of the script:
+
+- **script_name**: `str`. Label for the name of the script, which uniquely identifies it, and is delivered by an authoritative entity (such as ESGF, ENES, or WCRP)
+- **script_interface_version**: `str`. Label to specify the version of the script interface.
+- **mandatory_keys**: `list(str)`. List of reserved and custom keys for which the tool *must* provide values
+- **outputs**:`dict`. Dictionary of patterns with the purpose of allowing the tool to discover every output file and to assign a label to it. The patterns and the labels used as dictionary keys  can make use of keywords `variable`, `dataset` and `reference_dataset`. The tool may substitute the keywords with all possible values to form file basenames and test whether a corresponding file exists in the script's output directories. Dictionary values can be defined as pairs, in which the second element of the pair is a pattern to indicate the 'short_name' of the output variable.
+
+The following labels are accepted as optional settings, as certain tools may require their definition. Nevertheless, they can be ommitted if not relevant to the tool's configuration.
```

Comment:
Note that allowing optional parameters puts a burden on diagnostic developers, because they have to cater for the case where these are not provided in the input settings. This is a potential source of bugs in diagnostic scripts.

Reply:
There is a misunderstanding : this section (script formal description) is a static file, not a file that changes for every script all